### PR TITLE
NEXUS-21953: removed manifest file content from asset attributes

### PR DESCRIPTION
**Pull request by igorudovika**
NEXUS-21953: removed manifest file content from asset attributes

### CODE REVIEW

File: nexus-repository-conan-it/src/test/java/org/sonatype/nexus/plugins/conan/internal/ConanProxyIT.java [hunk 1]
```diff
@@ -182,8 +209,9 @@ public void retrievePackageWhenRemoteOnline() throws Exception {
 
   @Test
   public void retrieveInfoWhenRemoteOnline() throws Exception {
+    CloseableHttpResponse response = proxyClient.get(PATH_MANIFEST);
+    response.close();
```
Comment (mlukaretkyi) [inline nexus-repository-conan-it/src/test/java/org/sonatype/nexus/plugins/conan/internal/ConanProxyIT.java]: use try with resources or try, finally

File: nexus-repository-conan-it/src/test/java/org/sonatype/nexus/plugins/conan/internal/ConanProxyIT.java [hunk 2]
```diff
@@ -219,6 +247,16 @@ public void checkComponentRemovedWhenAssetRemoved() throws IOException {
     assertThat(component, is(equalTo(null)));
   }
 
+  @Test
+  public void failRetrieveConaninfoWhenWrongHashInConanmanifest() throws Exception {
+    CloseableHttpResponse response = proxyClient.get(LIB_WITH_WRONG_CONANINFO_HASH_DOWNLOAD_URLS_PATH);
+    response.close();
+    response = proxyClient.get(LIB_WITH_WRONG_CONANINFO_HASH_CONANMANIFEST_PATH);
+    response.close();
+    assertThat(status(proxyClient.get(LIB_WITH_WRONG_CONANINFO_HASH_CONANINFO_PATH)), is(HttpStatus.NOT_FOUND));
```
Comment (mlukaretkyi) [inline nexus-repository-conan-it/src/test/java/org/sonatype/nexus/plugins/conan/internal/ConanProxyIT.java]: use try with resources or try, finally

File: nexus-repository-conan-it/src/test/java/org/sonatype/nexus/plugins/conan/internal/ConanProxyIT.java (hunk 1 above)
Comment (igorudovika) [reply nexus-repository-conan-it/src/test/java/org/sonatype/nexus/plugins/conan/internal/ConanProxyIT.java]: I agree with you but we need close response before next request run. Without it we will face with test stuck. It looks like some restriction of testing env. Therefore with such approach I will need introduce empty block.
something like:
`try(CloseableHttpResponse response = proxyClient.get(PATH_MANIFEST)) {
    }` I also does not look good.

File: nexus-repository-conan-it/src/test/java/org/sonatype/nexus/plugins/conan/internal/ConanProxyIT.java (hunk 2 above)
Comment (igorudovika) [reply nexus-repository-conan-it/src/test/java/org/sonatype/nexus/plugins/conan/internal/ConanProxyIT.java]: Same as previous.

Comment (igorudovika) [reply nexus-repository-conan-it/src/test/java/org/sonatype/nexus/plugins/conan/internal/ConanProxyIT.java]: As we discussed I have added some comments to make test flow more clear.
e7d5a0b1c7948d6a8d7708f9a90b2388dee97648

File: nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/metadata/ConanManifest.java [hunk 3]
```diff
@@ -51,4 +52,14 @@ public static AttributesMap parse(TempBlob blob) {
     }
     return attributesMap;
   }
+
+
+  /**
+   * Extract all the md5 for files and add to attributes for later lookup
+   * @param blob
+   * @return
```
Comment (chernikovmaksym) [inline nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/metadata/ConanManifest.java]: Add description for the `return`

File: nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/metadata/ConanHashVerifier.java [hunk 4]
```diff
@@ -55,14 +55,14 @@ public HashCode lookupHashFromAsset(final StorageTx tx, final Bucket bucket, fin
     return null;
   }
 
-  private AttributesMap getAttributes(final StorageTx tx, final Bucket bucket, final String assetPath) {
+  private AttributesMap getConanmanifestHashes(final StorageTx tx, final Bucket bucket, final String assetPath) {
```
Comment (chernikovmaksym) [inline nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/metadata/ConanHashVerifier.java]: `getConanmanifestHashes` -> `getConanManifestHashes`

Comment (igorudovika) [reply nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/metadata/ConanHashVerifier.java]: I agree but there is conanmanifest.txt file in Conan world. So I will keep as it.

**Pull request merged**